packages/core/src/payment-integration/default-payment-integration-service.ts

```diff
@@ -49,6 +51,12 @@ export default class DefaultPaymentIntegrationService implements PaymentIntegrat
         this._storeProjection = this._storeProjectionFactory.create(this._store);
     }
 
+    async dispatch(action: Action) {
```

Comment:
This feels a bit odd, don't think we want to expose a generic dispatch method. Is there a way you can create a specific action needed? 

Another question is does core package need to know about this action at all? Can this be achieved by an API call in paypal package?

Reply:
I think it might be great to have such method to reduce amount of methods I payment integration package specific for each payment provider.

We can't go with API call in paypal package, because we need to update checkout state with data, that we get from paypal.

Reply:
I think the point of the service is to avoid such generic methods, may I ask what is the use case we are trying to solve. Which bit in state needs to be updated?

Reply:
We need to have an ability to update customers and instruments states with PayPal Connect profile data

Reply:
I am thinking on creating 2 different methods:
1) updateCustomer method, that will be responsible for updating customer state;
2) updateInstruments method, that will be responsible for updating vaulted instruments state;

In this case we will have 2 generic but meaningful methods and anyone will understand how to use them. What do you think @animesh1987?

Reply:
@serhii-tkachenko yeah, we should avoid just exposing dispatch to the payment-integrations-service. I believe `payment-integrations-service` it's like an inner SDK, so adding 2 different methods seems like a better idea to me.

Reply:
Ok, I will close the PR because we should find another solution for that